setup.py

```diff
@@ -142,6 +144,7 @@ def _set_builtin(name, value):
     [udunits_ext] = cythonize(
         udunits_ext,
         compiler_directives=COMPILER_DIRECTIVES,
+        # Assert python 3 source syntax. Default for Cython>=3, but let's be definite.
```

Comment:
My main motive is to suppress a warning, rather than 'belt and braces'. Would you be OK mentioning that?

Reply:
Sure, that's still a sufficient reason. https://github.com/SciTools/cf-units/pull/460/commits/f2012517e4a35376b8c2b364652ea5a417319dbd